src/main/java/com/avaya/letscode/Cart.java

```diff
@@ -7,11 +7,11 @@
 
 public class Cart {
     //List of items in the cart with respective unit quantity
-    Map<Product, CartItem> items;       
+    private Map<Product, CartItem> items;       
     
     //List of offers applicable to the cart
     // Map<String, Offer> cartOffers;
-    List<Offer> cartOffers;
+    private List<Offer> cartOffers;
```

Comment:
This can also be made final as above.

Reply:
Also, can we call this "offers" as opposed to cartOffers. We are already inside Cart object.